scalyr_agent/builtin_monitors/syslog_monitor.py

```diff
@@ -18,6 +18,10 @@
 from __future__ import absolute_import
```

Comment:
It would be good to add a docstring to this module to explain to the developers how request processing works and that we use a thread pool for processing requests and reading data from the socket.